### ISSUE TITLE
Support resolutions > 800x600

### ISSUE THREAD

**Issue by stefanhendriks** [stefanhendriks/Dune-II---The-Maker#56]
After #49 the next challenge is to support multiple resolutions.

This will break a few things still, because the in-game UI is fixed, see the `gfxinter.dat` file.

**Comment by stefanhendriks** [comment]
Looking in the git history, quit a bit of work could get cherry-picked perhaps to speed up this process:

https://github.com/stefanhendriks/Dune-II---The-Maker/commits/rewrite-sfml?after=e26ab5772815274d2352ad35c2e3eec0644ef718+734

**Comment by stefanhendriks** [comment]
It might be even sufficient to diff between master and this https://github.com/stefanhendriks/Dune-II---The-Maker/tree/db954212caf60e903d5e383a6e3b71fabf48ada8 commit now. It is WIP 352 (?) which should support multiple resolutions.

**Comment by stefanhendriks** [comment]
Use this https://github.com/stefanhendriks/Dune-II---The-Maker/compare/latest-wip-before-messing-up

and compare with master. Cherry-pick commits wherever needed.

**Comment by stefanhendriks** [comment]
Closing this in favor of #49
